Error when building the project from git

The following error occurs  when building the project using "meson compile -C _build"
All previous steps ran well
My Os is Manjaro Gnome Stable up to date (gnome 41.1)

```rust
Compiling libadwaita v0.1.0-beta-1
error[E0061]: this function takes 2 arguments but 1 argument was supplied
   --> src/ui/widgets/logged_in/library/image_stack.rs:220:33
    |
220 |                     self_.stack.scroll_to(&image);
    |                                 ^^^^^^^^^ ------ supplied 1 argument
    |                                 |
    |                                 expected 2 arguments
    |
note: associated function defined here
   --> /mnt/E/Apps_Sources/Epic-Asset-Manager/_build/target/cargo-home/registry/src/github.com-1ecc6299db9ec823/libadwaita-0.1.0-beta-1/src/auto/carousel.rs:160:12
    |
160 |     pub fn scroll_to<P: IsA<gtk::Widget>>(&self, widget: &P, animate: bool) {
    |            ^^^^^^^^^

error[E0061]: this function takes 2 arguments but 1 argument was supplied
   --> src/ui/widgets/logged_in/library/image_stack.rs:231:33
    |
231 |                     self_.stack.scroll_to(&image);
    |                                 ^^^^^^^^^ ------ supplied 1 argument
    |                                 |
    |                                 expected 2 arguments
    |
note: associated function defined here
   --> /mnt/E/Apps_Sources/Epic-Asset-Manager/_build/target/cargo-home/registry/src/github.com-1ecc6299db9ec823/libadwaita-0.1.0-beta-1/src/auto/carousel.rs:160:12
    |
160 |     pub fn scroll_to<P: IsA<gtk::Widget>>(&self, widget: &P, animate: bool) {
    |            ^^^^^^^^^

For more information about this error, try `rustc --explain E0061`.
error: could not compile `epic_asset_manager` due to 2 previous errors
[3/4] Generating data/io.github.achetagames.epic_as...nager.metainfo.xml_data_merge with a custom command
FAILED: src/epic_asset_manager 
/bin/bash /mnt/E/Apps_Sources/Epic-Asset-Manager/build-aux/cargo.sh /mnt/E/Apps_Sources/Epic-Asset-Manager/_build /mnt/E/Apps_Sources/Epic-Asset-Manager src/epic_asset_manager '' epic_asset_manager
ninja: build stopped: subcommand failed.
```

--- a/src/ui/widgets/logged_in/library/image_stack.rs
+++ b/src/ui/widgets/logged_in/library/image_stack.rs
@@ -217,7 +217,7 @@ impl EpicImageOverlay {
             clone!(@weak self as image_stack => move |_, _| {
                 let self_: &imp::EpicImageOverlay = imp::EpicImageOverlay::from_instance(&image_stack);
                 if let Some(image) = self_.stack.nth_page((self_.stack.position().round() as u32) + 1) {
-                    self_.stack.scroll_to(&image);
+                    self_.stack.scroll_to(&image, true);
                 };
             })
         );
@@ -228,7 +228,7 @@ impl EpicImageOverlay {
             clone!(@weak self as image_stack => move |_, _| {
                 let self_: &imp::EpicImageOverlay = imp::EpicImageOverlay::from_instance(&image_stack);
                 if let Some(image) = self_.stack.nth_page((self_.stack.position().round() as u32).saturating_sub(1)) {
-                    self_.stack.scroll_to(&image);
+                    self_.stack.scroll_to(&image, true);
                 };
             })
         );

